src/main/java/uk/gov/pay/products/resources/ProductResource.java

```diff
@@ -155,4 +156,13 @@ public Response findProductByProductPath(
                 .map(product -> Response.status(OK).entity(product).build())
                 .orElseGet(() -> Response.status(NOT_FOUND).build());
     }
+
+    @GET
+    @Path("/stats/products")
+    @Produces(APPLICATION_JSON)
+    public Response findProductsAndStats(@QueryParam("gatewayAccountId") Integer gatewayAccountId) {
+        logger.info(format("Listing usage stats on all non-prototype payment links for gateway account [gateway_account=%s]", gatewayAccountId));
```

Comment:
(Minor) We have standardised key for gateway account id logging https://github.com/alphagov/pay-java-commons/blob/master/logging/src/main/java/uk/gov/pay/logging/LoggingKeys.java#L73